src/coreclr/nativeaot/System.Private.CoreLib/src/System/Delegate.cs

```diff
@@ -43,7 +43,7 @@ protected Delegate([DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.Al
 
         internal object m_firstParameter;
         internal object m_helperObject;
-        internal IntPtr m_extraFunctionPointerOrData;
+        internal nint m_extraFunctionPointerOrData;
```

Comment:
This is another case where `void*` is probably "more appropriate".

Reply:
I was thinking about that initially, but this field was actually used for 2 purpose, function pointer in one case, and as `invocationCount`  in other case, so made it `nint`.

Reply:
Making this into `void*` takes lot more change and eventually used in public APIs that accept/return IntPtr and needs cast back, seems better to leave as is